bats/tests/helpers/paths.bash

```diff
@@ -100,6 +100,7 @@ if is_windows; then
     PATH_DISTRO="$PATH_DATA/distro"
     PATH_DISTRO_DATA="$PATH_DATA/distro-data"
     PATH_EXTENSIONS="$PATH_DATA/extensions"
+    SYSTEM32="/mnt/c/Windows/System32"
```

Comment:
We should not need this. If we did, we would have to translate it from `%SystemRoot%\System32`.